diff -up zig-0.14.0-20250205/CMakeLists.txt.omv~ zig-0.14.0-20250205/CMakeLists.txt
--- zig-0.14.0-20250205/CMakeLists.txt.omv~	2025-02-13 06:28:42.044213901 +0100
+++ zig-0.14.0-20250205/CMakeLists.txt	2025-02-13 06:29:00.634409266 +0100
@@ -893,7 +893,7 @@ set_target_properties(zig2 PROPERTIES
   LINK_FLAGS "${ZIG2_LINK_FLAGS}"
 )
 target_include_directories(zig2 PUBLIC stage1)
-target_link_libraries(zig2 LINK_PUBLIC zigcpp)
+target_link_libraries(zig2 LINK_PUBLIC zigcpp z zstd)
 
 if(MSVC)
   target_link_libraries(zig2 LINK_PUBLIC ntdll.lib ws2_32.lib)
