diff -up zig/CMakeLists.txt.omv~ zig/CMakeLists.txt
--- zig/CMakeLists.txt.omv~
+++ zig/CMakeLists.txt
@@ -708,7 +708,7 @@ set_target_properties(zig2 PROPERTIES
   LINK_FLAGS "${ZIG2_LINK_FLAGS}"
 )
 target_include_directories(zig2 PUBLIC stage1)
-target_link_libraries(zig2 LINK_PUBLIC zigcpp)
+target_link_libraries(zig2 LINK_PUBLIC zigcpp z zstd)
 
 if(MSVC)
   target_link_libraries(zig2 LINK_PUBLIC ntdll.lib ws2_32.lib)
